scripts/test/test-cli.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+export REGISTRY=${REGISTRY:-$USER}
+export REPO_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../.." && pwd )"
+export PORTER_HOME=${PORTER_HOME:-$REPO_DIR/bin}
+# Run tests in a temp directory
+export TEST_DIR=/tmp/porter/terraform
+mkdir -p ${TEST_DIR}
+pushd ${TEST_DIR}
+trap popd EXIT
+
+# Copy terraform assets
+cp -r ${REPO_DIR}/build/testdata/bundles/terraform/terraform .
+
+# Copy in the terraform porter manifest
+cp ${REPO_DIR}/build/testdata/bundles/terraform/porter.yaml .
+
+${PORTER_HOME}/porter build
+
+${PORTER_HOME}/porter install --debug --param file_contents='foo!'
+
+echo "Verifying instance output(s) via 'porter instance outputs list' after install"
+list_outputs=$(${PORTER_HOME}/porter instance outputs list)
+echo "${list_outputs}"
+echo "${list_outputs}" | grep -q "file_contents"
+echo "${list_outputs}" | grep -q "foo!"
+
+${PORTER_HOME}/porter invoke --action=plan --debug
+
+${PORTER_HOME}/porter upgrade --debug --param file_contents='bar!'
+
+echo "Verifying instance output(s) via 'porter instance output show' after upgrade"
+${PORTER_HOME}/porter instance output show file_contents | grep -q "bar!"
+
+cat ${PORTER_HOME}/claims/terraform.json
```

Comment:
Heads up that this is going to break as soon as claims is redone. Once it's merged, we should switch to `porter installation show terraform`

Reply:
@carolynvs oh good call.  Will the other `instance` commands break then as well?  (Like `porter instance output show` above).  Or will there be a deprecation period for `instance` commands once `installation` is introduced?

Reply:
I'll merge for now and we can tend to the cli tests once needed; or perhaps we'll be ready to convert to integration tests and api usage instead.